terraform/ratingservice/02_cloudsql.tf

```diff
@@ -27,6 +27,7 @@ resource "random_password" "db_password" {
 }
 
 resource "google_sql_database_instance" "rating_service" {
+  project             = var.gcp_project_id
```

Comment:
nit: this one and following additions are due to delete of the `ratingservice/01_provider.tf`. Would it be less trouble and less error prone keeping the provider declaration instead of introducing the need for this line in every google's resource?

Reply:
I had to remove the provider from the module, because terraform doesn't support providers when using `count` in modules.

There may be another way to set provider globally if you think this is an anti-pattern. I'd have to look into it